packages/destination-actions/src/destinations/salesforce/account/index.ts

```diff
@@ -11,6 +11,12 @@ const action: ActionDefinition<Settings, Payload> = {
   fields: {
     operation: operation,
     traits: traits,
+    externalIdFieldName: {
+      label: 'External ID field name',
+      type: 'string',
+      required: true,
+      description: 'The name of the field that will be used to identify the account in Salesforce.'
```

Comment:
Suggest something like this

```suggestion
      description: 'API Name of the External ID field on Account'
```

https://help.salesforce.com/s/articleView?id=000325076&type=1

Reply:
I think we can probably remove this though since we'll want customers to use the "Record Matchers" field and input the external ID there.